docs/build/api/get_offerings/get_offerings.md

```diff
@@ -0,0 +1,539 @@
+---
+sidebar_position: 2
+---
+
+# Finding compute resources on the Marketplace
+
+The Fluence compute marketplace is a decentralized platform where you can find and rent compute resources from various providers worldwide. Each provider has its own offers in different data centers with different configurations and prices. The marketplace API makes it easy to discover resources that meet your specific requirements.
+
+In this guide, we'll walk through how to:
+
+1. Search for available compute offers
+2. Filter resources based on your requirements
+3. Understand and compare different provider offerings
+4. Select the optimal resources for your needs
+
+## Searching for Available Offers
+
+The marketplace offers a powerful API endpoint that allows you to search for available compute resources with specific filters:
+
+```bash
+POST https://api.fluence.dev/marketplace/offers
+```
+
+### Understanding search parameters (constraints)
+
+You can use the request body to filter offers based on your specific requirements. All filters are optional, you can use them to find whether there are offers that match your requirements.
+
+**Example of a request body with all filters applied:**
+
+```json
+{
+  "basicConfiguration": "cpu-8-ram-16gb-storage-25gb",
+  "additionalResources": {
+    "ramMb": 2048,
+    "storage": [
+      {
+        "type": "NVMe",
+        "megabytes": 20480
+      }
+    ]
+  },
+  "hardware": {
+    "cpu": [
+      {
+        "manufacturer": "AMD",
+        "architecture": "Zen" // TODO: add brand and generation
+      }
+    ],
+    "memory": [
+      {
+        "type": "DDR",
+        "generation": "5"
+      }
+    ],
+    "storage": [
+      {
+        "type": "NVMe"
+      }
+    ]
+  },
+  "datacenter": {
+    "countries": ["FR"] // TODO: maybe add requirements for data center certifications
+  },
+  "maxTotalPricePerEpochUsd": "12.57426"
+  // TODO: add amount of instances (with additional resources)
```

Comment:
kill them todos.

Reply:
would be nice to annotate required, optional and wildcard arguments in the examples

Reply:
maybe reconcile with https://vodopad.mainnet.fluence.dev/swagger-ui/#/offers/ .. is that a different method? there' go to be more to that than available_locations.